mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ConvertScreenV2.tsx

```diff
@@ -0,0 +1,355 @@
+        </ThemedTextV2>
+
+        <ConversionLabel sourceUnit={sourceToken.unit} targetUnit={targetToken.unit} />
+
+        <View style={tailwind('w-full flex-col mt-6')}>
+          <ThemedSectionTitleV2
+            testID='convert_title'
+            text={translate('screens/ConvertScreen', 'I WANT TO CONVERT')}
+          />
+          <TransactionCard
+            maxValue={new BigNumber(sourceToken.amount)} status={transactionCardStatus}
+            onChange={onPercentagePress}
+            containerStyle={tailwind('border-t-0.5')}
+          >
+            <TransactionCardWalletTextInputV2
+              inputType='numeric'
+              displayClearButton={amount !== ''}
+              displayFocusStyle
+              onChangeText={setAmount}
+              value={amount}
+              hasBottomSheet={false}
+              onFocus={() => setIsInputFocus(true)}
+              onBlur={() => setIsInputFocus(false)}
+              onClearButtonPress={() => setAmount('')}
+              inputContainerStyle={tailwind('pb-5 pt-3 px-0')}
+              placeholder='0.00'
+              testID='convert_input'
+            />
+          </TransactionCard>
+          <ThemedTextV2
+            style={tailwind('font-normal-v2 text-xs px-5 pt-2')}
+            light={tailwind('text-mono-light-v2-500', {
+              'text-red-v2': hasError,
+              'text-orange-v2': showMaxUTXOWarning && !hasError
+            })}
+            dark={tailwind('text-mono-dark-v2-500', {
+              'text-red-v2': hasError,
+              'text-orange-v2': showMaxUTXOWarning && !hasError
+            })}
+            testID='source_balance'
+          >
+            {
+              translate('screens/ConvertScreen', hasError
+                ? 'Insufficient balance'
+                : showMaxUTXOWarning
+                  ? 'A small amount of UTXO is reserved for fees'
+                  : 'Available: {{amount}} {{unit}}', {
+                amount: new BigNumber(sourceToken.amount).toFixed(8),
+                unit: getDisplayUnit(sourceToken.unit)
+              })
+            }
+          </ThemedTextV2>
+        </View>
+
+        {amount.length > 0 && (
+          <View style={tailwind('flex-col w-full')}>
+            <ConversionResultCard
+              unit={getDisplayUnit(targetToken.unit)} oriTargetAmount={targetToken.amount}
+              totalTargetAmount={BigNumber.maximum(new BigNumber(targetToken.amount).plus(convAmount), 0).toFixed(8)}
+            />
+
+            <ThemedTextV2
+              style={tailwind('font-normal-v2 text-xs text-center pt-12 pb-5')}
+              light={tailwind('text-mono-light-v2-500')} dark={tailwind('text-mono-dark-v2-500')}
+            >
+              {`${translate('screens/ConvertScreen', 'Review full details in the next screen')}`}
+            </ThemedTextV2>
+          </View>
+        )}
+
+        <View style={tailwind('w-full px-7', { 'mt-56': amount.length === 0 })}>
```

Comment:
This can help to address the UI shifting a bit when `amount` changes.

But to me the ideal UI would be when there is no `amount`, we dont hardcode any margin, but `justify-between` in outermost `<ThemedSrollView>`
```suggestion
        <View style={[tailwind('w-full px-7'), { 'marginTop': amount.length === 0 ? 229 : 0 }]}>
```

Reply:
done, adjusted margin top

Reply:
^ my second sentence is optional, probably need to check designer too if they agree

Reply:
re-adjusted layout by using `justify-between` for 'Confirm' button display